test/unit/test_intrahost.py

```diff
@@ -152,7 +152,7 @@ def test_single_strand_bias_hard_filter(self):
         self.assertEqual(output[0][7:], expected[7:])
 
 
-@unittest.skipIf(tools.is_osx(), "vphaser2 osx binary from bioconda has issues")
+#@unittest.skipIf(tools.is_osx(), "vphaser2 osx binary from bioconda has issues")
```

Comment:
did the osx version get fixed?

Reply:
It seems to be? The test passes on OSX and I can get help text when I call it in isolation. Do you recall what the issue was?